public/index.html

```diff
@@ -6,9 +6,12 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
-      name="description"
-      content="Web site created using create-react-app"
+      name="Karan Asthana"
```

Comment:
It won't work. Meta tags are for SEO and it won't recognise this name "Karan Asthana". Name is the prop which can have values like description, keywords, author etc.

Reply:
If you want you can write Karan Asthana in content only, but name should be description in your case.